### PR TITLE
test: cover supported node versions with travis

### PR DESCRIPTION
**Pull request by rvagg**
I think this might be the best way to get full coverage in travis. @cclauss @joaocgreis

### CODE REVIEW

**Comment by cclauss** [comment]
Sure.  We can work this.  A lot of this was done on https://github.com/refack/GYP

File: .travis.yml [hunk 1]
```diff
@@ -18,6 +16,18 @@ matrix:
...
+      python: 3.7
+      before_install: nvm install 8
+    - name: "Node.js 10 & Python 3.7 on Linux"
+      python: 3.7
+      before_install: nvm install 10
+    - name: "Node.js 12 & Python 3.7 on Linux"
+      python: 3.7
+      before_install: nvm install 12
   allow_failures:
     - os: windows
```
Comment (cclauss) [inline .travis.yml]: Perhaps add __python: 3.7__ to __allow_failures__ ;-)

**Comment by cclauss** [comment]
While you are in here, can you please change __F72__ on line 39 to __F7__ so that we add the [__flake8 F70x tests](http://flake8.pycqa.org/en/latest/user/error-codes.html) to our scans?  I know that we do not have any __F70x__ issues today but putting this in place would catch any new occurrences.

**Comment by rvagg** [comment]
> please change F72 on line 39 to F7
gibberish! but done

**Comment by cclauss** [comment]
Do you understand why Node 6 fails?  Is it worth fixing?

**Comment by rvagg** [comment]
yes, it will turn magically green when #1808 is merged

**Pull request closed without merging**